### PR TITLE
nfc: Fix long frame NFC Reader timeout

### PR DESCRIPTION
**Pull request by KAGA164**

### CODE REVIEW

File: lib/st25r3911b/st25r3911b_nfca.c [hunk 1]
```diff
@@ -71,7 +71,7 @@ LOG_MODULE_DECLARE(st25r3911b);
 /* NFC-A CRC initial value. NFC Forum Digital 2.0 6.4.1.3. */
 #define NFCA_CRC_INITIAL_VALUE 0x6363
 
-#define RXS_TIMEOUT 10
+#define RXS_TIMEOUT 19776UL
```
Comment (kapi-no) [inline lib/st25r3911b/st25r3911b_nfca.c:74]: Am I understanding it correctly? This timeout value is in milliseconds so it seems that the new timeout value is ~20s.

Comment (kapi-no) [reply lib/st25r3911b/st25r3911b_nfca.c:74]: Is it necessary to use such a big timeout to handle long packets correctly?

Comment (kapi-no) [reply lib/st25r3911b/st25r3911b_nfca.c]: Now, it looks much better :)

Comment (KAGA164) [reply lib/st25r3911b/st25r3911b_nfca.c]: I set this now to 200ms. I mistook this value with the FDT.

**Comment by kapi-no** [comment]
@joerchan, could you merge it?

**Comment by joerchan** [comment]
@KAGA164 There are build errors, can you rebase this so we can get a green CI?

**Pull request merged**